node/hack/governor/src/index.ts

```diff
@@ -246,7 +246,7 @@ axios
       // The token symbol has changed
       // We take a substring of the symbol to cut the "=" character we added above
       else if (tokenParts[0] + "-" + tokenParts[1] + "-" + newTokenSymbol.substring(1) != token) {
-        changedSymbols.push(token + "->" + newTokenSymbol);
+        changedSymbols.push(token + "->" + newTokenSymbol.substring(1));
```

Comment:
Are the changes to this script supposed to be in  this PR?

Reply:
Yep I noticed the output was missing (and had the extra `=` char) so fixed that as part of this PR